pkg/build/controller/build/overrides/overrides.go

```diff
@@ -84,6 +89,22 @@ func (b BuildOverrides) ApplyOverrides(pod *v1.Pod) error {
 		pod.Annotations[k] = v
 	}
 
+	// Override Tolerations
+	if len(b.config.Tolerations) != 0 {
+		glog.V(5).Infof("Overriding tolerations for pod %s/%s", pod.Namespace, pod.Name)
+		pod.Spec.Tolerations = []v1.Toleration{}
+		for _, toleration := range b.config.Tolerations {
+			t := v1.Toleration{}
+
+			if err := kapiv1.Convert_core_Toleration_To_v1_Toleration(&toleration, &t, nil); err != nil {
```

Comment:
@deads2k is this an ok way to convert from the internal toleration object to external?

Reply:
yes